plugins/generator-1.20.1/datapack-1.20.1/jsontriggers/recipe_unlocked.json.ftl

```diff
@@ -0,0 +1,6 @@
+"${registryname}_${cbi}": {
+  "trigger": "minecraft:recipe_unlocked",
+  "conditions": {
+    "recipe": "${input$recipe}"

```

Comment:
This input requires recipe registry name, not item registry name, so the input item will not work reliably, especially not with custom items or complex recipes.

I also see lack of mapping system for MCItem, was this tested?

Reply:
I did test it with Items, yes

But i might've tested too "secure", as in, i did make sure all items had a Recipe before crafting.

I shall modify it to have a Recipe Input